common/common.go

```diff
@@ -195,6 +206,22 @@ func GetFilterFields() []string {
 	return filterFields
 }
 
+func GetRenameFields() map[string]string {
+	if renameFields == nil {
+		renameFields = getFieldMappings("RENAME_FIELDS")
+	}
+
+	return renameFields
+}
+
+func GetAliasFields() map[string]string {
+	if aliasFields == nil {
+		aliasFields = getFieldMappings("ALIAS_FIELDS")
```

Comment:
I believe we'll want to add this option to the templates where it can be used. Since we're at least looking at ALB, see the RenameFields option for a guide in [in this `aws-alb-logs.yml` template](https://github.com/honeycombio/agentless-integrations-for-aws/blob/main/templates/aws-alb-logs.yml).